plugin.js

```diff
@@ -159,6 +168,12 @@ function casPlugin (server, options) {
       username: session.username,
       attributes: session.attributes
     }
+    if (_options.value.sessionCredentialsMappings) {
+      const dotProp = require('dot-prop')
+      for (let mapping of _options.value.sessionCredentialsMappings) {
```

Comment:
Huh? Not as of https://github.com/jsumners/hapi-cas/pull/7/commits/de720b887308767ba84b3a889c998b7ab0dc8af5 ; https://github.com/charlesread/hapi-cas/blob/master/plugin.js#L172